target/linux/generic/files/drivers/net/phy/rtl8367b.c

```diff
@@ -1634,21 +1634,36 @@ static struct rtl8366_smi_ops rtl8367b_smi_ops = {
 };
 
 #ifdef CONFIG_OF
-static const struct of_device_id rtl8367b_match[] = {
-	{ .compatible = "realtek,rtl8367b" },
+struct rtl8367xx_chip {
+	unsigned long flags;
+};
+
+const struct rtl8367xx_chip rtl8367b_chip = {
+	.flags = 0,
+};
+
+const struct rtl8367xx_chip rtl8367c_chip = {
+	.flags = RTL8367B_FLAG_MATCH_CHIP_NUM,
+};
+
+static const struct of_device_id rtl8367bc_match[] = {
```

Comment:
rtl8367bc_match name makes no sense. Use rtl8367b_match as it is the module name.